### PR TITLE
Starter Gear

### PR DESCRIPTION
**Pull request by feend78**
This addresses Issue #20. (Also #39)

Summary
- Updated/added typeclasses: `Item`, `Equippable`, `Weapon`, `TwoHandedWeapon`, `RangedWeapon`, `TwoHandedRanged`
- Getting/dropping items affects encumbrance and movement traits on the character.
- Equipping weapons and armor affect attack and defense traits on the character.
- Updated and simplified the EquipHandler.
- Refactored equipping functionality into "equip" command, which will accept any item.
- Also implemented weapon- and armor-specific "wield" and "wear" aliases.
- Added an overridden version of the "inventory" command that excludes equipped items.
- Created a series of prototypes files in the world/content directory and modified the settings.py config file accordingly.
- Created prototypes for a decent sampling of weapons and armor from the OA rulebooks.
- Docs and tests

The selection of items was based mostly on which effects/abilities they had. Nearly all weapons and armor with no special abilities or effects are represented, as well as many of the +x Damage or +x Defense items. Combat abilities and effects for items have not been implemented in this branch, and are being intentionally left until combat system details are better resolved.

Based on the IRC conversation this evening, I wanted to just submit this version which implements all of "equip", "wield", and "wear" commands as a starting point.

### CODE REVIEW

File: world/content/prototypes_weapons.py [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+    "damage": 1,
+}
+
+MAUL_HAMMER = {
+    "key": "a maul hammer",
+    "aliases": ["maul hammer", "maul", "hammer"],
+    "typeclass": "typeclasses.weapons.TwoHandedWeapon",
+    "desc": "A heavy mass of metal mounted atop a sturdy staff, "
+            "ready to deliver crushing blows to your enemies."
+            "to wield it.",
```
Comment (ergodicbreak) [inline world/content/prototypes_weapons.py]: typo?

**Comment by ergodicbreak** [comment]
Thank you @feend78 for the hefty contribution! It looks good, I'll leave it open for @whitenoiseoss if he has any comments.

**Comment by feend78** [comment]
Fixed the typo you found.

I'm also wondering if @kylemecklem might like to look this over too. I had already chosen the weapons and armor that I've included in this branch when I became aware of his document for Ainneve starter info. I'd welcome his comments or suggestions on the item descriptions in the prototypes files.

**Comment by feend78** [comment]
Also, I know @whitenoiseoss wanted the EquipHandler to be removed, but I did end up keeping it. I see it as providing extended validation, ordering of slots, and a more streamlined syntax within commands than manipulating the char.db.slots dict directly. It also afforded the opportunity to re-introduce limbs, which disappeared a couple of branches ago. They currently only affect the order that slots are displayed by the "equip" command in Ainneve, but could be a basis for more complex armor slot systems for others.

**Comment by kylemecklem** [comment]
@feend78 wow great job with the weapons, armor and equipment! It should be said, though, that certain weapons/armor were "balanced" by giving them weapon abilities. For example, a dagger only costs 30 copper coins, deals 1 damage and weighs one-half weight. A scythe has the same stats (except weighing 1 weight) but costs over three times as much (1 silver coin). The difference is that the dagger has no special abilities while the scythe has "hook", "sweep" and "reach 1".

I made a set of starting gear for the warrior, scout and arcanist. Some of the gear may not be what you're looking for in terms of a MUD (such as an orb, fishing pole & tackle and chalk); but I figured all that really means is the quest designers have to be creative in how those items get implemented. For example maybe there will be a puzzle involving the shifting of identical stone walls; and the character with chalk can mark & identify which wall moved to where. Here's the starting equipment for the three archetypes:

https://docs.google.com/document/d/10C6YXdniCYWOX0HlYtR5RW2EBlsqscJBCJnjVr_-0WQ/

Since you seem to have the equipment covered, I'll begin work on making combat-useful spells that fit within the tier level needed for the arcanist, as well as a multitude of enemy NPCs that player characters can battle.

**Comment by feend78** [comment]
Hi @kylemecklem,

I agree that abilities and effects help to balance the items, but since they are related more closely to the combat system, they just haven't been implemented yet. There is another issue related to the combat system, and I'd be interested in having a conversation there about the specific item abilities we want to implement, as the two OA rulebooks described in the Ainneve wiki seem to list different abilities for the same starter items.

It shouldn't be a problem to add some of the other items that you mentioned as starting equipment too. For now, they can be created as instances of the `Item` typeclass. I had been planning to handle starter gear in Ainneve similarly to how it's described in the OA rulebooks--by rolling and giving characters a number of SC, then letting them "buy" what they want from among the items. Would you want your items to be just given to players based on archetype, or would they be listed as items to buy at this stage of character generation?

Also, since you mentioned spells here, I was wondering how Level 0 Arcanists will be handled? I don't see any Tier 0 spells in the OA manual, and it seems a bummer to have to play through to level 1 without any magic as an Arcanist.

**Comment by kylemecklem** [comment]
@feend78 the equipment in the basic rulebook should be used over anything listed in the complete rulebook. The complete rulebook hasn't been worked on in a long time; but acts as a roadmap for what will be developed for the future books (basic, intermediate, expert and master).

In the basic rulebook, on page 23 (as listed in the pages) there is a "universal" rule pertaining to equipment use (lock picks, rope, flint & steel, compass, etc.). It states that equipment offers one or more of the following functions:
- Allows a specific action to be attempted (e.g. lock pick allows a lock to be picked) OR the equipment allows for the player to make two rolls instead of one (when making a skill test) and choose the more favorable roll. This encourages players to purchase and use mundane items throughout their expedition.
- Some items have special abilities or effects
- Most items have a worth or value in coin that players can use to purchase or sell items

I think allowing the players to purchase their own equipment is a great idea. If so, you can disregard my "starting equipment" idea; as that was just an attempt on my behalf to pre-generate the equipment buying process.

I'll begin work on some level 0 spells and fill in the missing spells for the tier 3 black magic slots.

**Pull request merged**